packages/tools/src/tools/TrackballRotateTool.ts

```diff
@@ -172,6 +170,7 @@ class TrackballRotateTool extends BaseTool {
     const { rotateIncrementDegrees } = this.configuration;
     const enabledElement = getEnabledElement(element);
     const { viewport } = enabledElement;
+    this._reduceSampleDistance(viewport);
```

Comment:
I'm not entirely certain about the reason for invoking the `reduceSampleDistance` function during each drag operation. This function executes multiple times, potentially tens of times, within a single interaction.

Ideally, it should start before the mouse event (preMouseDown) and reset after the mouseUp event.